src/components/movie-card/movie-card.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC } from 'react';
+import styles from './movie-card.module.scss';
+
+interface MovieCardProps {
+  img: string;
+  title: string;
+  year: number;
+  description: string;
+}
+export const MovieCard: FC<MovieCardProps> = ({
```

Comment:
Advise you to always separate your interfaces and components with empty rows to avoid code overlapping